soda/core/soda/execution/check/check.py

```diff
@@ -236,6 +235,7 @@ def get_cloud_dict(self):
             "location": self.check_cfg.location.get_cloud_dict(),
             "dataSource": self.data_source_scan.data_source.data_source_name,
             "table": Partition.get_table_name(self.partition),
+            "datasets": [Partition.get_table_name(self.partition)],
```

Comment:
This property `datasets` is not supported by backend API is not supported by backend and it will be ignored

Reply:
removed